cmd/osm-bootstrap/crds/policy_retry.yaml

```diff
@@ -0,0 +1,100 @@
+# Custom Resource Definition (CRD) for OSM's policy specification.
+#
+# Copyright Open Service Mesh authors.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: retries.policy.openservicemesh.io
+  labels:
+    app.kubernetes.io/name : "openservicemesh.io"
+spec:
+  group: policy.openservicemesh.io
+  scope: Namespaced
+  names:
+    kind: Retry
+    listKind: RetryList
+    shortNames:
+      - retry
+    singular: retry
+    plural: retries
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              required:
+                - source
+                - destinations
+                - retryPolicy
+              properties:
+                source:
+                  description: Source the Retry policy is applicable to.
+                  type: object
+                  required:
+                    - kind
+                    - name
+                    - namespace
+                  properties:
+                    kind:
+                      description: Kind of this source.
+                      type: string
+                    name:
+                      description: Name of this source.
+                      type: string
+                    namespace:
+                      description: Namespace of this source.
+                      type: string
+                destinations:
+                  description: Destinations the sink that the Retry policy is applicable to.
+                  type: object
+                  required:
+                    - kind
+                    - name
+                    - namespace
+                  properties:
+                    kind:
+                      description: Kind of this source.
+                      type: string
+                    name:
+                      description: Name of this source.
+                      type: string
+                    namespace:
+                      description: Namespace of this source.
+                      type: string
+                retryPolicy:
+                  description: Destinations the sink that the Retry policy is applicable to.
```

Comment:
what does the sink imply ?

Reply:
changed to: "description: Retry policy that will be applied to the source and destination services"